webhook/cert/manager.go

```diff
@@ -297,10 +296,10 @@ func (m *Manager) writeCertOnDisk(ctx context.Context) error {
 		return err
 	}
 
-	if err := ioutil.WriteFile(filepath.Join(m.CertDir, m.CertName), cert, 0666); err != nil {
+	if err := os.WriteFile(filepath.Join(m.CertDir, m.CertName), cert, 0600); err != nil {
```

Comment:
is the 0666 -> 0600 change intended?

Reply:
The issue has found by gosec. We create folder with `0700`, so i think we should create files with `0600`. FYI Operator passed e2e tests with this change.